eng/CHECKENFORCER

```diff
@@ -28,7 +28,7 @@ message: >
   /check-enforcer evaluate
   ```
   
-  Typically evaulation only takes a few seconds. If you know that your pull
+  Typically evaluation only takes a few seconds. If you know that your pull
```

Comment:
@mitchdenny is it OK to modify this check enforcer file like this? Do other places/repos need to be fixed?